libresapi/src/api/TransfersHandler.h

```diff
@@ -5,16 +5,20 @@
 
 #include <retroshare/rsfiles.h>
 #include <retroshare/rspeers.h>
+#include <retroshare/rsnotify.h>
 
 namespace resource_api
 {
 
-class TransfersHandler: public ResourceRouter, Tickable
+class TransfersHandler: public ResourceRouter, Tickable, NotifyClient
 {
 public:
-	TransfersHandler(StateTokenServer* sts, RsFiles* files, RsPeers *peers);
+	TransfersHandler(StateTokenServer* sts, RsFiles* files, RsPeers *peers, RsNotify* notify);

```

Comment:
Every time ti is possible it is better to use a reference instead of a pointer, in this case i would prefer if you use a reference at least for notify +RsNotify& notify+, it seems possible to use references for the other parameters too but use a reference instead of pointers when possible at least on new code